ui/appui-react/src/appui-react/widget-panels/Frontstage.tsx

```diff
@@ -553,10 +553,11 @@ export function initializePanel(
       maxSize,
     });
 
+  const collapsed = panelDef.defaultState !== StagePanelState.Open;
```

Comment:
What is a stagepanel in a "Popup" state (last enum value)? I dont see that used anywhere in the code ? (And, it would mean that when in this stage, the panel is collapsed, which would sound unexpected...) Should the enum value be marked as _deprecated_ ?